capa/features/freeze/__init__.py

```diff
@@ -208,6 +337,18 @@ class Config:
         allow_population_by_field_name = True
 
 
+class DynamicFeatures(BaseModel):
+    global_: Tuple[GlobalFeature, ...] = Field(alias="global")
+    file: Tuple[FileFeature, ...]
+    processes: Tuple[ProcessFeatures, ...]
+
+    class Config:
+        allow_population_by_field_name = True
+
+
+Features: TypeAlias = Union[StaticFeatures, DynamicFeatures]
```

Comment:
Thus far, there's one `Freeze` class for both the static and dynamic freeze, and the type of the freeze (static/dynamic) is determined based on the type of its extractor instance (static or dynamic) or the type of its features instance (`StaticFeatures` or `DynamicFeatures`). Should we create two separate freeze instances `StaticFreeze` and `DynamicFreeze`?

Thus far I've used one class (`Freeze`) to represent both `StaticFreeze` and `DynamicFreeze` since the only difference between the two is the types of some of their attributes. There are no extra attributes (like with `StaticFeatures` and `DynamicFeatures` for example) or whatever...